spotify.py

```diff
@@ -0,0 +1,192 @@
+from datetime import datetime, timedelta
+import time
+from spotify_secrets import DEVICE_ID, CLIENT_ID, CLIENT_SECRET, REFRESH_TOKEN
```

Comment:
This will create an Error, there is no spotify_secrets module in our repo. Update the conda environment if you imported new modules.